File: SignatureRequests/SignatureRequests.Core/RequestObjects/FormRequest.cs
using SignatureRequests.Core.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SignatureRequests.Core.RequestObjects
{
    public class FormRequest
    {
        public int Id { get; set; }
        public string FilePath { get; set; }

        public string Title { get; set; }

        public string Description { get; set; }

        public DateTime CreateDate { get; set; }

        public virtual UserEntity User { get; set; }

        public int UserId { get; set; }
    }
}

File: SignatureRequests/SignatureRequests.Webs/Scripts/react/src/Entities/FormRequest.ts

class FormRequest {
  filePath: string;
  title: string;
  description? : string;
  createDate: Date;
  userId: number;
  numPages: number;

  constructor(data: any) {
    this.filePath = data.FilePath;
    this.title = data.Title;
    this.description = data.Description;
    this.createDate = data.CreateDate;
    this.userId = data.UserId;
    this.numPages = data.NumPages;
  }
}

export default FormRequest;
